OAuth/ResourceOwner/FacebookResourceOwner.php

```diff
@@ -35,6 +35,18 @@ class FacebookResourceOwner extends GenericOAuth2ResourceOwner
     /**
      * {@inheritDoc}
      */
+    public function getUserInformation(array $accessToken, array $extraParameters = array())
+    {
+        if (isset($this->options['appsecret_proof']) && $this->options['appsecret_proof']) {
```

Comment:
no need to check with ``isset``. There is a default value in the OptionsResolver, so it will always be set